tests/security/audit/auditctl.pm

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Controlling the Audit system using auditctl
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772 tc#1768551
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $audit_rules = '/etc/audit/rules.d/audit.rules';
+    my $audit_log = '/var/log/audit/audit.log';
+    select_console 'root-console';
+
+    # Disable audit
+    validate_script_output('auditctl -e 0', sub { m/enabled 0/ });
+
+    # Clear audit log, generate audit reports and these should not be recorded
+    assert_script_run("echo -n '' > $audit_log");
+    systemctl('stop apparmor');
+    systemctl('start apparmor');
+    if ([-s $audit_log]) {
+        record_info('Auditd disabled: ', 'No audit reports have been recorded');
```

Comment:
`-s  File has nonzero size (returns size in bytes).`
the message in record_info is opposite.

Reply:
`[]` is not mandatory.

Reply:
Here audit log has been cleared so recorded information of audit log should be empty (or more precisely 'null' since I'm using `echo -n` here). So I think it is corret that `-s audit_log` should fail here and return 1 and it suggests the log is indeed empty.